iConverter/Pages/Transaction/Presenter/TransactionPresenter.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  TransactionPresenter.swift
+//  iConverter
+//
+//  Created by Aram Semerjyan on 13.04.22.
+//
+
+final class TransactionPresenter: BasePresenter {
+    weak var vc: TransactionViewController?
+
+    var viewModel: TransactionViewModel? { vc?.viewModel }
```

Comment:
I'd personally won't do this and just directly call `vc?.viewModel` but if you'll make it like this, you should declare as private